public/app/plugins/datasource/elasticsearch/datasource.test.ts

```diff
@@ -895,7 +895,6 @@ describe('ElasticDatasource', function (this: any) {
 
     const interpolatedQuery = ds.interpolateVariablesInQueries([query], {})[0];
 
-    expect(interpolatedQuery.query).toBe('*');
```

Comment:
this was removed in favor of a dedicated test in https://github.com/grafana/grafana/pull/42060/files#diff-4c45fed23a4567bef120a187afc98ade9787d19c61fdef3e9cde0382017ea024R741